lib/module.js

```diff
@@ -95,38 +96,81 @@ export default function SentryModule (moduleOptions) {
     }
   }
 
+  const apiMethods = await getBrowserApiMethods()
+
+  if (options.lazy) {
+    const defaultLazyOptions = {
+      injectMock: true,
+      injectLoadHook: false,
+      mockApiMethods: true,
+      chunkName: 'sentry',
+      webpackPrefetch: false,
+      webpackPreload: false
+    }
+
+    options.lazy = Object.assign({}, defaultLazyOptions, options.lazy)
+
+    if (!options.lazy.injectMock) {
+      options.lazy.mockApiMethods = []
```

Comment:
A combination of `lazy: true` and server-side sentry being disabled would result in server-side not mocking the methods. Kinda unlikely case but still unexpected.

(I'll looking into making `mockApiMethods` a root option so this is just to not forget the issue.)

Reply:
@rchl Thanks man, really appreciate your help with this pr. Hope you don't mind it, but as you are opinionated about a lot of things (which is definitely a good thing!) it could be the fastest then trying to explain to me what you mean exactly :)

:bowing_man:

Reply:
I've misunderstood how it works. I thought that "lazy" logic will affect what is mocked for the non-lazy mocks. But it's not the case so I'll just abandon my idea of making `mockApiMethods` a root option.